src/samples/Event.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef __BASE_SAMPLES_EVENT_HH
+#define __BASE_SAMPLES_EVENT_HH
+
+#include <base/Time.hpp>
+
+namespace base {
+namespace samples {
+
+    struct Event
+    {
+        Event():x(0),y(0),ts(::base::Time::fromSeconds(0)),polarity(false){}
+        Event(unsigned int x_, unsigned int y_, ::base::Time ts_, uint8_t p_)
+            :x(x_),y(y_),ts(ts_),polarity(p_){}
+
+        unsigned int x;
+        unsigned int y;
```

Comment:
Try to avoid `int` or `unsigned int`. Use fixed-size versions `uintXX_t`. `int` changes size on different architectures (ARM)